app/hotels/src/map/MapScreen.js

```diff
@@ -9,9 +9,9 @@ import MapView from './MapView';
 import HotelSwipeList from './HotelSwipeList';
 import type { MapScreen as MapScreenData } from './__generated__/MapScreen.graphql';
 
-type Props = {|
+type Props = {
   data: MapScreenData,
-|};
+};
```

Comment:
This is probably not intentional, is it?

Reply:
cc @jaroslav-kubicek

Reply:
no, fixed